src/components/blog/blog-grid.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import BlogCard from './blog-card'
+import { ChevronDown } from 'react-feather'
+
+const BlogGrid = () => {
+  return (
+    <div className='container' id='blog-grid'>
+      <div className='row'>
+        <BlogCard />
+        <BlogCard />
+        <BlogCard />
+        <BlogCard />
+        <BlogCard />
+        <BlogCard />
+        <BlogCard />
+        <BlogCard />
+        <BlogCard />
+        <div className='col-12 d-flex justify-content-center'>
+          <a href='/sign-up-node-operator/'>
```

Comment:
The link here is wrong

Reply:
This link should be fixed in the blog PRs. Just let me know how that/those PRs look. Thanks!